internal/actions/sync_branch.go

```diff
@@ -139,52 +139,71 @@ func syncBranchRebase(
 		return nil, errors.WrapIff(err, "failed to get head of branch %q", branch.Name)
 	}
 
-	if branch.IsStackRoot() {
-		trunk := branch.Parent.Name
-		if !opts.ToTrunk {
+	parent := branch.Parent
```

Comment:
[minor] Can you make this `parentState` for disambiguation from `parentBranch`? I think it'll make it slightly easier to read below.

Reply:
Done.